src/libertem/web/server.py

```diff
@@ -79,6 +79,12 @@ def make_app(event_registry, shared_state):
             "state": shared_state,
             "event_registry": event_registry
         }),
+        (r"/api/shutdown/", ShutdownHandler, {
+            "state": shared_state,
+            "event_registry": event_registry
+        }
+
+        ),
```

Comment:
Style nitpick: closing parenthesis can be on the same line as closing curly brace.